langlib/ballerina-java/src/main/ballerina/Ballerina.toml

```diff
@@ -2,12 +2,3 @@
 org-name = "ballerina"
 version = "0.9.0"
 
-[platform]
-target = "java8"
-
-    [[platform.libraries]]
-    artifactId = "java"
-    version = "0.9.0"
-    path = "./lib/ballerina-java-0.9.0-java.jar"
-    groupId = "ballerina"
-    modules = ["java"]
```

Comment:
We don't need this section anymore with jdk 11?

Reply:
It is not there in any of the lang toml files. They are there in stdlib toml files only